modules/sdk-coin-ada/src/lib/transaction.ts

```diff
@@ -99,6 +113,47 @@ export class Transaction extends BaseTransaction {
       });
     }
 
+    if (this._transaction.body().certs()) {
+      for (let i = 0; i < this._transaction.body().certs()!.len(); i++) {
+        const cert = this._transaction.body().certs()!.get(i);
+        if (cert.as_stake_registration() !== undefined) {
```

Comment:
Same question here about breaking the loop on assignment.

Reply:
yeah, same as above, not huge reason to break